File: problemas/1000/1020.cs
using System;

namespace uri1020
{
    internal static class Program
    {
        private static void Main()
        {
            int dias = int.Parse(Console.ReadLine());
            Console.WriteLine($"{dias / 365} ano(s)");
            dias %= 365;
            Console.WriteLine($"{dias / 30} mes(es)");
            dias %= 30;
            Console.WriteLine($"{dias} dia(s)");
        }
    }
}

File: problemas/1000/1020.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
        var sc = new Scanner(System.in);

        int dias = sc.nextInt();
        System.out.printf("%s ano(s)\n", dias / 365);
        dias %= 365;
        System.out.printf("%s mes(es)\n", dias / 30);
        dias %= 30;
        System.out.printf("%s dia(s)\n", dias);
    }
}